onchain/src/quests/template_quest.cairo

```diff
@@ -0,0 +1,69 @@
+use art_peace::templates::interfaces::{
+    ITemplateStoreDispatcher, ITemplateStoreDispatcherTrait, ITemplateVerifierDispatcher,
+    ITemplateVerifierDispatcherTrait, TemplateMetadata
+};
+
+#[starknet::contract]
+pub mod TemplateQuest {
+    use starknet::{ContractAddress, get_caller_address};
+    use art_peace::{IArtPeaceDispatcher, IArtPeaceDispatcherTrait};
+    use art_peace::quests::{IQuest, QuestClaimed};
+
+    #[storage]
+    struct Storage {
+        art_peace: IArtPeaceDispatcher,
```

Comment:
It might be better to store this as a `ContractAddress` and have a separate storage slot for `ITemplateStoreDispatcher` since you won't be using the `IArtPeaceDispatcher` for anything.

Reply:
you mean having `art_peace: ContractAddress` ?

Reply:
Yes, that's right